charts/ziti-controller/values.yaml

```diff
@@ -184,6 +184,10 @@ image:
   command: ["ziti", "controller", "run"]
   # -- args for the entrypoint command
   args: ["{{ include \"configMountDir\" . }}/ziti-controller.yaml"]
+  # -- additional arguments can be passed directly to the container to modify ziti runtime arguments
+  additionalArgs: []
+  # -- additional environment variables to pass into runtime
+  env: []
```

Comment:
This default value for controller env can be empty/null (not defined, not ""), which implies `{}` (not `[]`). It's because this is a dictionary, not an array/list. 

The router's env value is at top level, not nested under image, so it's probably best for them to be consistent. I could see moving the controller's env value to match if you feel strongly they both belong under image. 

The controller deployment template currently references top level `.Values.env` ( not nested under `image`).

Reply:
I'm not crazy about changing the nesting unless there's a pretty good reason. I'll match this to the router setup as suggested.